src/svgmaker.py

```diff
@@ -0,0 +1,49 @@
+import os
+import random
+from typing import List
+
+import svgwrite
+
+from src.color_table import Color, ColorTable
+
+DATA_DEFAULT_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data')
+W = 140
+H = 40
+
+
+def create_svg(color_groups: List[List['Color']]):
+    with open(os.path.join(DATA_DEFAULT_PATH, "test.svg"), mode='w') as file:
+        dwg = svgwrite.Drawing(size=(f'{W * len(color_groups[0])}px',
+                                     f'{H * len(color_groups)}px'
+                                     ))
+        for group, y in zip(color_groups, range(0, len(color_groups) * H, H)):
+            for x, color in enumerate(group):
+                rect = dwg.rect(insert=(x * W, y), size=(W, H), fill=color.hex)
+                dwg.add(rect)
+                text = dwg.text(list(map(int, color.lab_values)),
```

Comment:
list(map(int, color.lab_values)) -> [int(v) for v in color.lab_values]

Reply:
done